.clang-tidy

```diff
@@ -2,52 +2,50 @@
 Checks: >
   clang-diagnostic-*,
   clang-analyzer-*,
-  -clang-analyzer-deadcode.DeadStores,
   bugprone-*,
+  -bugprone-narrowing-conversions,
+  -bugprone-easily-swappable-parameters,
+  -bugprone-assignment-in-if-condition,
+  -bugprone-casting-through-void,
   cert-*,
   -cert-err58-cpp,
+  -cert-dcl50-cpp,
   concurrency-*,
   cppcoreguidelines-*,
   -cppcoreguidelines-avoid-do-while,
-  misc-*,
-  modernize-*,
-  -modernize-use-trailing-return-type,
-  performance-*,
-  -performance-avoid-endl,
-  portability-*,
-  readability-*,
-  -readability-function-cognitive-complexity,
-  -misc-const-correctness,
-  -misc-include-cleaner,
-  -readability-identifier-length,
   -cppcoreguidelines-pro-type-union-access,
   -cppcoreguidelines-pro-type-reinterpret-cast,
   -cppcoreguidelines-avoid-non-const-global-variables,
-  -misc-use-anonymous-namespace,
-  -readability-avoid-nested-conditional-operator,
   -cppcoreguidelines-pro-bounds-constant-array-index,
   -cppcoreguidelines-narrowing-conversions,
-  -bugprone-narrowing-conversions,
-  -bugprone-easily-swappable-parameters,
   -cppcoreguidelines-pro-bounds-pointer-arithmetic,
   -cppcoreguidelines-avoid-c-arrays,
-  -modernize-avoid-c-arrays,
   -cppcoreguidelines-pro-type-const-cast,
-  -bugprone-assignment-in-if-condition,
-  -performance-enum-size,
-  -bugprone-casting-through-void,
   -cppcoreguidelines-macro-usage,
   -cppcoreguidelines-pro-bounds-array-to-pointer-decay,
-  -misc-non-private-member-variables-in-classes,
-  -readability-convert-member-functions-to-static,
-  -performance-unnecessary-value-param,
-  -readability-use-anyofallof,
   -cppcoreguidelines-pro-type-vararg,
+  -cppcoreguidelines-rvalue-reference-param-not-moved,
+  misc-*,
+  -misc-const-correctness,
+  -misc-include-cleaner,
+  -misc-use-anonymous-namespace,
+  -misc-non-private-member-variables-in-classes,
   -misc-no-recursion,
-  -cert-dcl50-cpp,
+  modernize-*,
+  -modernize-use-trailing-return-type,
+  -modernize-avoid-c-arrays,
+  performance-*,
+  -performance-avoid-endl,
+  -performance-enum-size,
+  -performance-unnecessary-value-param,
   -performance-no-int-to-ptr,
-  -clang-analyzer-optin.cplusplus.VirtualCall,
-  -cppcoreguidelines-rvalue-reference-param-not-moved
+  portability-*,
+  readability-*,
+  -readability-function-cognitive-complexity,
+  -readability-identifier-length,
+  -readability-avoid-nested-conditional-operator,
+  -readability-convert-member-functions-to-static,
+  -readability-use-anyofallof
```

Comment:
There are lots of changes to the options here. Is this just sorting? Or are there semantic changes as well?

Reply:
It's just sorting.